backend/app/engine/projects/rewards/__init__.py

```diff
@@ -26,21 +27,24 @@ def __iter__(self):
 @dataclass
 class ProjectRewardsPayload:
     matched_rewards: int = None
-    allocations: List[AllocationItem] = None
+    before_allocations: List[AllocationItem] = None
```

Comment:
if its previous allocations, we need to name name this way `user_previous_allocations`

Reply:
this naming is this way because there are not specific to the user but for allocations made for current AW in general; makes sense to you?

Reply:
makes sense